react/src/hooks/utils.tsx

```diff
@@ -10,10 +10,12 @@ import {
     UseQueryOptions,
     UseQueryResult,
 } from "react-query";
-import { Query, SetDataOptions } from "react-query";
+import { Query } from "react-query";
 import { stringToBoolean } from "../functions";
 import { QueryWithSearchOptions as MTQueryWithSearchOptions } from "../typings";
 
+type SetDataOptions = NonNullable<Parameters<QueryClient["setQueryData"]>[2]>;
+
```

Comment:
What's going on here in the frontend? I think you might have an old version of react-query installed. Try running `yarn` again to update.

Reply:
Cool! Will try installing yarn again.